apps/frontend/src/app/2_molecules/WalletIdentity/WalletIdentity.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import classNames from 'classnames';
+
+import { Dropdown, DropdownSize } from '@sovryn/ui';
+
+import styles from './WalletIdentity.module.css';
+import { AddressBadge } from './components/AddressBadge/AddressBadge';
+import { DisconnectSubmenu } from './components/DisconnectSubmenu/DisconnectSubmenu';
+
+export type WalletIdentityProps = {
+  onDisconnect: () => void;
+  address: string;
+  className?: string;
+  dataActionId?: string;
+  hideSubmenu?: boolean;
+};
+
+export const WalletIdentity: FC<PropsWithChildren<WalletIdentityProps>> = ({
```

Comment:
I see no `children` used in component
```suggestion
export const WalletIdentity: FC<WalletIdentityProps> = ({
```